lib/mbta_v3_api/stream/instance.ex

```diff
@@ -131,7 +131,7 @@ defmodule MBTAV3API.Stream.Instance do
 
     consumer_subscribers =
       if consumer_dest do
-        Registry.count_match(MBTAV3API.Stream.PubSub, consumer_dest, :_)
+        Registry.lookup(MBTAV3API.Stream.PubSub, consumer_dest)
```

Comment:
tiny add in while looking at the existing health logs. I've noticed some timeouts while reporting these health logs:
```
62db153750ed 21:18:03.911 [error] GenServer #PID<0.2135.0> terminating
62db153750ed ** (stop) exited in: :sys.get_state(#PID<0.3383.0>)
62db153750ed ** (EXIT) time out
62db153750ed (stdlib 5.1.1) sys.erl:340: :sys.send_system_msg/2
62db153750ed (stdlib 5.1.1) sys.erl:139: :sys.get_state/1
62db153750ed (mobile_app_backend 0.1.0) lib/mbta_v3_api/stream/instance.ex:104: MBTAV3API.Stream.Instance.stage_health/1
62db153750ed (mobile_app_backend 0.1.0) lib/mbta_v3_api/stream/instance.ex:85: MBTAV3API.Stream.Instance.check_health/1
62db153750ed (mobile_app_backend 0.1.0) lib/mbta_v3_api/stream/supervisor.ex:34: anonymous fn/2 in MBTAV3API.Stream.Supervisor.log_health/0
62db153750ed (elixir 1.15.7) lib/enum.ex:2510: Enum."-reduce/3-lists^foldl/2-0-"/3
62db153750ed (mobile_app_backend 0.1.0) lib/mbta_v3_api/stream/supervisor.ex:33: MBTAV3API.Stream.Supervisor.log_health/0
62db153750ed (mobile_app_backend 0.1.0) lib/mbta_v3_api/stream/health.ex:19: MBTAV3API.Stream.Health.handle_info/2
```

Since we only care about the registry key, lookup should be faster than scanning to match the full registry.